### PR TITLE
Add Oku Trade to exchanges list

### LINKED ISSUE #191: Update exchange page
### Description

Oku DEX now supports LSK

### Location

https://docs.lisk.com/user/exchanges

### Additional Information
https://lisk.com/blog/posts/oku-is-live-on-lisk

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #191

### CODE REVIEW

File: docs/docs-user/exchanges.md [hunk 1]
```diff
@@ -69,11 +69,14 @@ Please note that using the wrong network/blockchain when depositing funds to an
 
 ## Decentralized Exchanges (DEX)
 
+| Name                          | Token pairs                                 |
+| :---------------------------- | :------------------------------------------ |
+| [Oku.Trade](https://oku.trade/app/lisk/trade/0xac485391eb2d7d88253a7f1ef18c37f4242d1a24?quoteToken=0x05D032ac25d322df992303dCa074EE7392C117b9) | LSK/USDT, LSK/USDC, LSK/ETH, ETH/USDT, etc. |
```
Comment (TalhaMaliktz) [inline docs/docs-user/exchanges.md]: For ease and synchronicity, we should add hyperlinks to the coin pairs in a way similar to how they are done throughout the page.

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

File: docs/docs-user/exchanges.md (hunk 1 above)
Comment (Tschakki) [reply docs/docs-user/exchanges.md]: Didn't see them - If you can find these links, we can add them.

Comment (TalhaMaliktz) [reply docs/docs-user/exchanges.md]: Yeah, I couldn't find them either. All good 👍🏼

Comment (TalhaMaliktz) [inline docs/docs-user/exchanges.md]: There is no `LSK/USDC` pair, I think!
image

Comment (Tschakki) [reply docs/docs-user/exchanges.md]: updated 👍

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 👍🏼 !

**Pull request merged**